lib/index.js

```diff
@@ -60,9 +60,16 @@ function update() {
 
 function _getJS(polyfill) {
   return request(polyfill.url).then(function (response) {
-    assert(response.status === 200)
-    return response.text()
-  }).then(function (js) {
+    var text;
+    try {
+       assert(response.status === 200, polyfill.name + ': ' + response.status);
```

Comment:
The real issue here is if a polyfill not returning a 200 then we wont get it.
And that might cause the users code to break further down.
So we need to catch it in some way and alert the user that a polyfill is missing.